couchexport/util.py

```diff
@@ -45,6 +45,10 @@ def clear_attachments(schema_or_doc):
         del schema_or_doc['_attachments']
     if schema_or_doc and 'case_attachments' in schema_or_doc:
         del schema_or_doc['case_attachments']
+    if schema_or_doc:
```

Comment:
yeah, unless a form had all three of those properties, which would be pretty surprising/rare.